src/helper/order.js

```diff
@@ -0,0 +1,70 @@
+import {getSelectedRootItems} from './selection';
+
+const bringToFront = function (onUpdateSvg) {
+    const items = getSelectedRootItems();
+    for (const item of items) {
+        item.bringToFront();
+    }
+    onUpdateSvg();
+};
+
+const sendToBack = function (onUpdateSvg) {
+    const items = getSelectedRootItems();
+    for (let i = items.length - 1; i >= 0; i--) {
+        items[i].sendToBack();
+    }
+    onUpdateSvg();
+};
+
+const bringForward = function (onUpdateSvg) {
+    const items = getSelectedRootItems();
+    // Already at front
+    if (items.length === 0 || !items[items.length - 1].nextSibling) {
+        return;
+    }
+
+    const nextSibling = items[items.length - 1].nextSibling;
+    for (let i = items.length - 1; i >= 0; i--) {
+        items[i].insertAbove(nextSibling);
+    }
+    onUpdateSvg();
+};
+
+const sendBackward = function (onUpdateSvg) {
+    const items = getSelectedRootItems();
+    // Already at front
+    if (items.length === 0 || !items[0].previousSibling) {
+        return;
+    }
+
+    const previousSibling = items[0].previousSibling;
+    for (const item of items) {
+        item.insertBelow(previousSibling);
+    }
+    onUpdateSvg();
+};
+
+const shouldShowSendBackward = function () {
```

Comment:
In the future, I think it is good practice to only submit code that is actually used in each PR. I know this code is very related, and will be used in follow-up PRs, but it complicates the git history to have features spread between different PRs. For example, it would make more sense to submit this code when the UI parts of enable/disable are also used. It leaves less unreachable code around in the git history.

Reply:
You're right, thanks. I have been being messy about bringing over the helper functions and then have to prune them later. I will do that in the future.